internal/weapons/catalyst/sacrificialjade/sacrificialjade.go

```diff
@@ -0,0 +1,87 @@
+package sacrificialjade
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/pkg/core"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+	"github.com/genshinsim/gcsim/pkg/core/keys"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/core/player/weapon"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+func init() {
+	core.RegisterWeaponFunc(keys.SacrificialJade, NewWeapon)
+}
+
+type Weapon struct {
+	Index    int
+	refine   int
+	c        *core.Core
+	char     *character.CharWrapper
+	lastSwap int
+	buff     []float64
+}
+
+func (w *Weapon) SetIndex(idx int) { w.Index = idx }
+
+func (w *Weapon) Init() error {
+	w.c.Tasks.Add(w.updateBuff(w.lastSwap), 6*60)
+	return nil
+}
+
+// When not on the field for more than 6s, Max HP will be increased by 32% and Elemental Mastery will be increased by 40.
+// These effects will be canceled after the wielder has been on the field for 6s.
```

Comment:
- Off-field timer for buff gain should be 5s and on-field timer for buff removal should be 10s.
- Please put these two timer values into separate consts so that it is consistent throughout the implementation. 
- You will most likely need 2 different tasks for these two things with src check:
    - On swapping out: change lastSwap, queue up off-field buff gain task in 5s.
    - On swapping in: change lastSwap, queue up on-field buff removal task in 10s.
- The tasks should not queue themselves up again.

Reply:
> Please put these two timer values into separate consts so that it is consistent throughout the implementation.

what means "separate consts"?